src/foremast/templates/configs/configs.json.j2

```diff
@@ -17,6 +16,7 @@
         "app_grace_period": 0,
         "max_inst": 3,
         "min_inst": 1,
+        "ssh_keypair": null,
```

Comment:
OK. Region is populated later. Possibly moving this to gogo-utils may help longer term.